liblouis/utils.c

```diff
@@ -136,25 +136,28 @@ _lou_charHash(widechar c) {
 	return (unsigned long int)c % HASHNUM;
 }
 
-char *EXPORT_CALL
-_lou_showString(widechar const *chars, int length) {
+const char *EXPORT_CALL
+_lou_showString(widechar const *chars, int length, int forceHex) {
 	/* Translate a string of characters to the encoding used in character
 	 * operands */
-	int charPos;
-	int bufPos = 0;
 	static char scratchBuf[MAXSTRING];
+	int bufPos = 0;
 	scratchBuf[bufPos++] = '\'';
-	for (charPos = 0; charPos < length && bufPos < (MAXSTRING - 2); charPos++) {
-		if (chars[charPos] >= 32 && chars[charPos] < 127)
-			scratchBuf[bufPos++] = (char)chars[charPos];
-		else {
+
+	for (int charPos = 0; (charPos < length) && (bufPos < (MAXSTRING - 2));
+			charPos += 1) {
```

Comment:
Why is `charPos += 1` better than `charPos++` ?

Reply:
> In theory, nothing. I don't like making a special case out of an increment/decrement by 1. I think it only makes coding sense to use ++ or -- when implementing a stack-like behaviour.

OK, fair enough. Fine with me. It's just that the rest of the code doesn't use that convention and that might make reading the code for an outsider more difficult.